build/Targets/VSL.Imports.targets

```diff
@@ -468,4 +468,18 @@
                IntermediatesDirectory="$(IntermediateOutputPath)"
                Type="$(SignType)" />
   </Target>
+
+  <!-- ====================================================================================
+
+  Copying vsixmanifest to a separate folder
+
+  ==================================================================================== -->
+
+  <Target Name="AfterBuild" Condition="'$(CreateVsixContainer)' == 'true'">
+     <Copy SourceFiles="source.extension.vsixmanifest" DestinationFolder="$(OutDir)\Manifests\$(MSBuildProjectName)" ContinueOnError="true" />
```

Comment:
Forgot to remove `ContinueOnError="true"`. The path is relative to the proj file. Would you prefer having a comment mentioning that?